app/uk/gov/hmrc/hmrcemailrenderer/templates/tcs/newTcsRenewalConfirmationMessageAlert.scala.html

```diff
@@ -1,24 +1,106 @@
-@(params: Map[String, Any])@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params,"Your tax credits renewal has been received", true, footerMessage = None, true) {
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Your tax credits renewal has been received", true, footerMessage = None, true) {
 
 <p style="font-size: 24px;line-height: 1.315789474; margin: 0 0 30px 0;">
-    Your online reference number is
+    Your reference number is
     <span style="white-space: nowrap;">@{params("renewalReference")}</span>
 </p>
 
-<h2>You do not need to do anything else</h2>
+<h2>What you can do next</h2>
+@if((params("isMultiple") == "true") && (params("anyEstimates") != "0") && (params("anyEstimates") != "1")){
+    <p style="font-size: 16px;line-height: 1.315789474;">You must:</p>
+    @if(((params("incompleteClaims") != "0") && (params("incompleteClaims") != "1"))){
+        <ul>
+            <li style="font-size: 16px;line-height: 1.315789474; margin: 0 0 10px 0;">complete your @{params("incompleteClaims")} remaining annual reviews to finish your renewal.</li>
+            <li style="font-size: 16px;line-height: 1.315789474; margin: 0 0 10px 0;">contact the tax credits helpline to tell us the actual amount for the estimates you gave by 31 January 2017.</li>
```

Comment:
31 of January 2017 is hard coded... is that right?
The same happens throughout the whole template.